### PR TITLE
Guide: Customizing Query Classes

### PR DESCRIPTION
**Pull request by CedricYii**
| Q | A |
| --- | --- |
| Is bugfix? | no |
| New feature? | no |
| Breaks BC? | no |
| Tests pass? | no |
| Fixed issues | #12462 |

Clearly separate classes' definition.
Add information for default query condition with init().
Suggest using andOnCondition() instead of andWhere() as it didn't work
with the left join example provided.

### CODE REVIEW

**Comment by CedricYii** [comment]
I just realized that I missread `$customers = Customer::find()->with('activeComments')->all();` is ok with andWhere().
But it wouldn't with `$customers = Customer::find()->joinWith('activeComments')->all();`

File: docs/guide/db-active-record.md [hunk 1]
```diff
@@ -1350,41 +1349,44 @@ class Comment extends ActiveRecord
         return new CommentQuery(get_called_class());
     }
 }
+````
+Now whenever you are performing a query (e.g. `find()`, `findOne()`) or defining a relation (e.g. `hasOne()`)
+with `Comment`, you will be calling an instance of `CommentQuery` instead of `ActiveQuery`.
 
+You can customize the `CommentQuery` class in many creative ways to improve your query building experience. For example,
+you can define new query building methods:
+````php
```
Comment (samdark) [inline docs/guide/db-active-record.md]: Need 3 ticks, not 4 for code blocks. Also there should be empty line before code.

Comment (CedricYii) [inline docs/guide/db-active-record.md]: Thank you, I didn't notice. Fixed.

**Comment by samdark** [comment]
Thanks!

**Pull request merged**